tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -427,6 +427,11 @@ namespace Datadog.Trace
         public SpanContextExtractor() { }
         public Datadog.Trace.ISpanContext? Extract<TCarrier>(TCarrier carrier, System.Func<TCarrier, string, System.Collections.Generic.IEnumerable<string?>> getter) { }
     }
+    public class SpanContextInjector
+    {
+        public SpanContextInjector() { }
```

Comment:
Technically this should have a public API metric too (which means you need to explicitly add the default constructor instead of using the implicit one)